app/src/test/kotlin/fr/free/nrw/commons/nearby/NearbyParentFragmentPresenterTest.kt

```diff
@@ -306,7 +306,7 @@ class NearbyParentFragmentPresenterTest {
         // 111.19 km real distance, return false if 148306.444306 >  currentLocationSearchRadius
         NearbyController.currentLocationSearchRadius = 148306.0
         val isClose = nearbyPresenter.searchCloseToCurrentLocation()
-        assertFalse(isClose!!.equals(false))
+        assertFalse(!isClose)
```

Comment:
So, removing the double negative, you mean
```suggestion
        assertTrue(isClose)
```

Reply:
Yes, that's right, I removed the double negative to make the assertion more straightforward and easier to read. It doesn't change the logic, just improves clarity.